### PR TITLE
added modal component.

### PR DESCRIPTION
**Pull request by zoyatanweer**

### CODE REVIEW

File: components/modal/modal.css [hunk 1]
```diff
@@ -0,0 +1,41 @@
+.modal{
+    background-color: #f8edeb;
```
Comment (zoef-yt) [inline components/modal/modal.css]: use var

File: components/modal/modal.css [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+}
+.modal-btn-primary,.modal-btn-secondary{
+    border-radius: 0.5rem;
+    font-size: 1.8rem;
+    padding: 0.5rem;
+    margin: 0.5rem;
+    cursor: pointer;
+}
+.modal-btn-secondary{
+    background-color: #f8edeb;
```
Comment (zoef-yt) [inline components/modal/modal.css]: use var

File: components/modal/modal.css [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    border: 2px solid var(--btn-secondary-hover-color);    
+}
+.modal-btn-primary{
+    background-color: #ff006e;
+    border: 2px solid #ff006e;
+    color: white;
```
Comment (zoef-yt) [inline components/modal/modal.css]: use var

**Review by zoef-yt — CHANGES_REQUESTED** [review]

File: components/modal/modal.css [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    margin-bottom: 1rem;
+    margin-top: 1rem;
+}
+.modal-dismiss-btn{
+    font-size: 2rem;
+    position: absolute;
+    left: 87rem;
+    top: 43rem;
+}
+.modal-about{
```
Comment (dev-prabhat) [inline components/modal/modal.css:22]: space between classes is irregular

File: components/modal/modal.html [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+  style="width: 774px; height: 391px; border:0; transform: scale(1); overflow:hidden;"
```
Comment (dev-prabhat) [inline components/modal/modal.html]: try to use external style

**Review by dev-prabhat — CHANGES_REQUESTED** [review]
code is clean , added some suggestions

**Review by Shubhamkr2610 — APPROVED** [review]
Just made the changes mentioned by other folks. Then It will also looks good to me and you then you are good to go

**Review by zoef-yt — APPROVED** [review]
Good to merge.

**Pull request merged**